Edits throughout manuscript (#17)
greenelab/iscb-diversity-manuscript: content/20.results.md

Because the concordance was high and the PubMed set was more complete, we used the PMC corresponding author information when it was available and the PubMed last author as a fallback when corresponding author information was missing.

### Assessing Gender Diversity of Authors and Honorees

Although Bioinformatics was established in 1998 and BMC Bioinformatics in 2000, the metadata for these journal papers before 2002 only have initials for first and/or middle author names.
Therefore, without first and middle names, we do not have author gender predictions before this year.

We observed a slow increase of the proportion of predicted female authors, arriving at just over 20% in 2019 (Fig. {@fig:gender_breakdown}A).
We observe very similar trend within each journal, but estimated female proportion has increased the least in _PLOS Computational Biology_ (Supplementary Fig. S1).
ISCB Fellows and keynote speakers appear to be more evenly split between men and women compared to the population of authors published in computational biology and bioinformatics journals (Fig. {@fig:gender_breakdown}B); however, it has not yet reached parity.

> An overall increasing female proportion trend appeared in each honor category, especially in the group of ISCB Fellows (Fig. {@fig:gender_breakdown}C), which markedly increased after 2015.

Reviewer: ```suggestion
We observed an increasing trend of honorees who were women in each honor category, especially in the group of ISCB Fellows (Fig. {@fig:gender_breakdown}C), which markedly increased after 2015.
```

### Assessing the Nationality Diversity of Authors and Honorees

We applied our Wiki2019-LSTM model to both our computational biology honorees dataset and our dataset of corresponding authors.
We found that the proportion of authors in the CelticEnglish categories had decreased (Fig. {@fig:region_breakdown}A), particularly for papers published in _Bioinformatics_ and _BMC Bioinformatics_ (Fig. {@fig:region_breakdown}B).
Among keynote speakers and fellows we found that the majority of honorees are predicted to be from CelticEnglish countries ({@fig:region_breakdown}C).
Though sample sizes were small, we did observe some differences between meetings.

> ISMB keynotes had more probability attributable to Israel, while RECOMB had more attributable to EastAsian countries ({@fig:region_breakdown}D).

Reviewer: I'm considering moving the busy panels to supplement, but I can put them back in if we think they're important for the main text.

Author: Ok - will go with this for now and then we can continue to revise and cite the supplementary figs instead if things move.
